concepticondata/conceptlists/Gabelentz-1892-1074.tsv

```diff
@@ -0,0 +1,1075 @@
+Gabelentz-1892-1074-1040	1040	Schiessen	to shoot	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	1172	SHOOT
+Gabelentz-1892-1074-1041	1041	Schlagen	to beat, to strike	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	2133	STRIKE OR BEAT
+Gabelentz-1892-1074-1042	1042	Schleifen, wetzen	to grind, to whet	VIII. Verba	F. Andere Verba (alphabetisch geordnet)		
+Gabelentz-1892-1074-1043	1043	Schneiden	to cut	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	1432	CUT
+Gabelentz-1892-1074-1044	1044	Spielen	to play	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	1413	PLAY
+Gabelentz-1892-1074-1045	1045	Stechen	to stab, to sting	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	1434	STAB
+Gabelentz-1892-1074-1046	1046	durchbohren	to pierce	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	398	PIERCE
+Gabelentz-1892-1074-1047	1047	stossen	to push	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	1452	PUSH
+Gabelentz-1892-1074-1048	1048	Stehlen	to steal	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	713	STEAL
+Gabelentz-1892-1074-1049	1049	Strafen	to punish	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	3649	PUNISH
+Gabelentz-1892-1074-1050	1050	Tätowiren	to tattoo	VIII. Verba	F. Andere Verba (alphabetisch geordnet)		
+Gabelentz-1892-1074-1051	1051	Tauschen	to swap, to exchange	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	2509	EXCHANGE
+Gabelentz-1892-1074-1052	1052	Theilen, vertheilen	to share	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	884	SHARE
+Gabelentz-1892-1074-1053	1053	Tragen	to carry	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	700	CARRY
+Gabelentz-1892-1074-1054	1054	Treffen	to hit a target	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	3307	HIT (A TARGET)
+Gabelentz-1892-1074-1055	1055	Treiben, vertreiben	to drive, to disperse	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	3290	IMPEL
+Gabelentz-1892-1074-1056	1056	Uebergeben, auslieferm	to hand over	VIII. Verba	F. Andere Verba (alphabetisch geordnet)		
+Gabelentz-1892-1074-1057	1057	Uebertreten	to pass over, to trespass	VIII. Verba	F. Andere Verba (alphabetisch geordnet)		
+Gabelentz-1892-1074-1058	1058	Verbergen	to hide	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	2486	HIDE
+Gabelentz-1892-1074-1059	1059	Verderben, verwüsten	to destroy	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	681	DESTROY
+Gabelentz-1892-1074-1060	1060	Vereinigen	to unite	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	1743	JOIN
+Gabelentz-1892-1074-1061	1061	Vergelten, rächen	to take revenge	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	495	REVENGE
+Gabelentz-1892-1074-1062	1062	Verletzen, verwunden	to wound someone, to hurt someone	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	2411	HURT (SOMEBODY)
+Gabelentz-1892-1074-1063	1063	Verlieren	to lose	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	1660	LOSE
+Gabelentz-1892-1074-1064	1064	Verrathen	to disclose something	VIII. Verba	F. Andere Verba (alphabetisch geordnet)		
```

Comment:
BETRAY?

Reply:
I didn't even think of this possible translation, interpreted this more as e.g. "ein Geheimnis verraten" :)
Should I add "betray" to the gloss and remove the mapping, then? I'm not sure whether we can discard one meaning here.

Reply:
True, perhaps unmap. The meaning BETRAY seemed like the more serious, less colloquial one, but who knows which one Gabelentz meant.